contracts/src/interfaces.cairo

```diff
@@ -1,2 +1,3 @@
 pub mod ierc20;
-pub mod ipeer_protocol;
\ No newline at end of file
+pub mod ipeer_protocol;
+pub mod ierc721;
```

Comment:
<img width="1512" alt="Screenshot 2024-11-24 at 05 49 44" src="https://github.com/user-attachments/assets/ce29da1c-f63c-470d-bfb7-4e5cb05d5b9b">
please fix so it builds

Reply:
@Dprof-in-tech I don't understand why your build is failing. Everything works fine on my end.